src/Microsoft.AspNetCore.SignalR.Core/IHubClients`T.cs

```diff
@@ -19,4 +19,12 @@ public interface IHubClients<T>
 
         T User(string userId);
     }
+
+    public interface IHubCallerClients<T> : IHubClients<T>
+    {
+        T Caller { get; }
+
+        T Others { get; }
```

Comment:
Do we have an issue tracking `OthersInGroup`? It's fine to not do that in this PR but it should be tracked :)

Reply:
Yup https://github.com/aspnet/SignalR/issues/734